<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardWidget</class>
 <widget class="QWidget" name="KeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget QPushButton
{
	height: 100%;
	background-color: rgb(252, 252, 252);
	border: none;
	font: 75 14pt &quot;Calibri&quot;;
}

#backspaceButton, #clearButton, #parentheseLeftButton, #parentheseRightButton, #addButton, #timesButton, #minusButton
{
	background-color: rgb(246, 246, 246)
}

#backspaceButton:hover, #clearButton:hover, #parentheseLeftButton:hover, #parentheseRightButton:hover, #addButton:hover, #timesButton:hover, #minusButton:hover
{
	background-color: rgb(220, 220, 220);
	border: 2px solid rgb(214, 214, 214);
}

#backspaceButton:pressed, #clearButton:pressed, #parentheseLeftButton:pressed, #parentheseRightButton:pressed, #addButton:pressed, #timesButton:pressed, #minusButton:pressed
{
	background-color: rgb(214, 214, 214);
}

#backspaceButton:disabled, #clearButton:disabled, #parentheseLeftButton:disabled, #parentheseRightButton:disabled, #addButton:disabled, #timesButton:disabled, #minusButton:disabled
{
	background-color: rgb(230, 230, 230);
	color: rgb(171, 171, 171);
}

QWidget QPushButton:hover
{
	background-color: rgb(220, 220, 220);
	border: 2px solid rgb(214, 214, 214);
}

QWidget QPushButton:pressed
{
	background-color: rgb(214, 214, 214);
}

QWidget *:disabled
{
	background-color: rgb(230, 230, 230);
	color: rgb(171, 171, 171);
}

#calculateButton 
{
	background-color: rgb(146, 194, 232)
}

#calculateButton:hover
{
	background-color: rgb(73, 157, 223);
	border: 2px solid rgb(69, 149, 212);
}

#calculateButton:pressed
{
	background-color: rgb(69, 149, 212)
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QPushButton" name="num1Button">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="num7Button">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="num8Button">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="num5Button">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="num2Button">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="dotButton">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="num4Button">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="num0Button">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="parentheseRightButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="parentheseLeftButton">
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="backspaceButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images/img.qrc">
       <normaloff>:/ico/backspace.png</normaloff>:/ico/backspace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="num9Button">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="num6Button">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="minusButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="num3Button">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="timesButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="divideButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>÷</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="calculateButton">
     <property name="text">
      <string>＝</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/img.qrc"/>
 </resources>
 <connections/>
</ui>
